modules/javafx.graphics/src/test/java/test/javafx/css/Node_cssStyleMap_Test.java

```diff
@@ -286,18 +291,6 @@ public void testRT_21212() {
 
     }
 
-    boolean containsProperty(CssMetaData key, Map<String,List<CascadingStyle>> map) {
-
-        if (map.containsKey(key)) return true;
-        List<CssMetaData> subProperties = key.getSubProperties();
-        if (subProperties != null && !subProperties.isEmpty()) {
-            for (CssMetaData subKey: subProperties) {
-                if (map.containsKey(subKey)) return true;
-            }
-        }
-        return false;
-    }
-
```

Comment:
I don't mind removing this method since it's unused, but probably as part of a "remove unused methods" PR.